### DISCUSSION TITLE
How to scan all targets within a folder?

### DISCUSSION THREAD

**Discussion by noob0x01** [projectdiscovery/nuclei#1450] [Q&A] [answered]
Hey,
example , i have a folder

test > domain1.txt , domain2.txt , domain3.txt 

how to scan full folder?

@ehsandeep bro

**Comment by ehsandeep** [comment, marked as answer]
You can merge all the inputs before passing to nuclei or use cat `cat 1.txt 2.txt 3.txt | nuclei -t cves`

**Reply by forgedhallpass** [reply]
If your folder contains only text files with URLs you want to scan, instead of listing every file separately (`cat 1.txt 2.txt 3.txt`), you can also use `cat *.txt`, or `cat domain*.txt` and pipe that to nuclei.
